pages/index.vue

```diff
@@ -34,6 +35,30 @@ export default {
       articles,
       categories
     }
+  },
+  computed: {
+    isDarkMode () {
+      if (process.browser) {
+        return window.matchMedia('(prefers-color-scheme: dark)').matches
+      }
+      return false
+    }
+  },
+  head () {
+    return {
+      title: 'BitcartCC Blog',
```

Comment:
There is no need to duplicate everything from `nuxt.config.js`, only favicon setting is enough.

Reply:
Removed the code since favicon with white background is enough